### DISCUSSION TITLE
Using the new update on rolling branch, how do you setup your typescript development?

### DISCUSSION THREAD

**Discussion by mandaputtra** [LunarVim/LunarVim#1695] [Q&A] [answered]
After the new braking change #1584, my typescript project doesn't have proper linting, formatting, and auto complete. Follow the documentation update like these.

```
lvim.lsp.override = { 'typescript', 'typescriptreact', 'javascript', 'javascriptreact' }

-- -- JAVASCRIPT TYPESCRIPT LSP
lvim.lang.typescript.formatters = { { exe = "prettier" } }
lvim.lang.typescriptreact.formatters = lvim.lang.typescript.formatters
lvim.lang.javascript.formatters = lvim.lang.typescript.formatters
lvim.lang.javascriptreact.formatters = lvim.lang.typescript.formatters

lvim.lang.typescript.linters = { { exe = "eslint" } }
lvim.lang.typescriptreact.linters = lvim.lang.typescript.linters
lvim.lang.javascript.linters = lvim.lang.typescript.linters
lvim.lang.javascriptreact.linters = lvim.lang.typescript.linters
```

It is successfully registering eslint and prettier but sometimes the tsserver aren't loaded. I already have tsserver installed. Is there any configuration that I need to do? I use rolling branch

![Screenshot from 2021-10-05 09-08-23](https://user-images.githubusercontent.com/23342943/135949139-fb33cf53-d91a-48b7-9e40-536b70ec19a9.png)

![Screenshot from 2021-10-05 09-08-35](https://user-images.githubusercontent.com/23342943/135949143-334348b5-d502-4715-99a9-5203bd07394c.png)

**Comment by kylo252** [comment, marked as answer]
Can you try `:LvimUpdate` again?

You can customize the root directory since c0e3c8d, and Prettier will only register for the specified filetype since f4899e3.

Let me know if you find any issues with the new workflow.
